build-tools-internal/src/main/resources/templates/breaking-changes.asciidoc

```diff
@@ -15,10 +15,10 @@ coming::[${majorDotMinorDotRevision}]
 [discrete]
 [[breaking-changes-${majorDotMinor}]]
 === Breaking changes
-<% if (breakingByNotabilityByArea.isEmpty()) { %>
+
 // tag::notable-breaking-changes[]
```

Comment:
If there are breaking changes, the opening `// tag::notable-breaking-changes[]` tag should be lower, on line 27 (after the "The following changes.." paragraph. See for example: https://github.com/elastic/elasticsearch/blob/main/docs/reference/migration/migrate_8_5.asciidoc?plain=1#L22

Reply:
Looking further down I see there is already a notable-breaking-changes tag. So this fix is not valid. The alternative explanation is that there are no breaking changes with the `notable` flag. We need an alternative fix.